sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Monitor.OpenTelemetry.Exporter.Internals.Platform;
+using Azure.Monitor.OpenTelemetry.LiveMetrics.Internals;
+using Azure.Monitor.OpenTelemetry.LiveMetrics;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OpenTelemetry;
+using OpenTelemetry.Logs;
+using System;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter
+{
+    /// <summary>
+    /// Extension methods for setting up Azure Monitor in an <see cref="OpenTelemetryBuilder" />.
+    /// </summary>
+    public static class OpenTelemetryBuilderExtensions
+    {
+        /// <summary>
+        /// Configures Azure Monitor Exporter for all signals.
+        /// </summary>
+        /// <param name="builder"><see cref="IOpenTelemetryBuilder"/>.</param>
+        /// <returns>The supplied <see cref="IOpenTelemetryBuilder"/> for chaining calls.</returns>
+        /// <remarks>
+        /// <para>
+        /// This method configures Azure Monitor for use with OpenTelemetry by adding the Azure Monitor exporter for logging,
+        /// distributed tracing, and metrics. It also configures the OpenTelemetry logger to include formatted messages and
+        /// parsed state values.
+        /// </para>
+        /// </remarks>
+        public static IOpenTelemetryBuilder UseAzureMonitorExporter(this IOpenTelemetryBuilder builder)
+        {
+            builder.Services.TryAddSingleton<IConfigureOptions<AzureMonitorExporterOptions>,
+                        DefaultAzureMonitorExporterOptions>();
+            return builder.UseAzureMonitorExporter(o => { });
+        }
+
+        /// <summary>
+        /// Configures Azure Monitor Exporter for logging, distributed tracing, and metrics.
+        /// </summary>
+        /// <param name="builder"><see cref="IOpenTelemetryBuilder"/>.</param>
+        /// <param name="configureAzureMonitor">Callback action for configuring <see cref="AzureMonitorExporterOptions"/>.</param>
+        /// <returns>The supplied <see cref="IOpenTelemetryBuilder"/> for chaining calls.</returns>
+        /// <remarks>
+        /// <para>
+        /// This method configures Azure Monitor for use with OpenTelemetry by adding the Azure Monitor exporter for logging,
+        /// distributed tracing, and metrics. It also configures the OpenTelemetry logger to include formatted messages and
+        /// parsed state values.
+        /// </para>
+        /// </remarks>
+        public static IOpenTelemetryBuilder UseAzureMonitorExporter(this IOpenTelemetryBuilder builder, Action<AzureMonitorExporterOptions> configureAzureMonitor)
```

Comment:
Should users be allowed to call this more than once? UseOtlpExporter will throw for repeated calls IIRC.

Reply:
`UseAzureMonitorExporter` will adhere to the principles from `UseOtlpExporter`. To keep the PR straightforward, I did not include those changes. In a follow-up PR, I will update the code to throw an exception if it is called more than once.